app/services/docker/pluginService.js

```diff
@@ -52,13 +53,5 @@ angular.module('portainer.services')
     return deferred.promise;
   };
 
-  service.volumePlugins = function(systemOnly) {
```

Comment:
I'd rather keep the `volumePlugins` and `networkPlugins` options.

I believe that the fix is only to add a `return` statement.

e.g.

```javascript
service.networkPlugins = function(systemOnly) {
  return servicePlugins(systemOnly, 'Network', 'docker.networkdriver/1.0');
};
```

Reply:
I dropped the previous fix and added this. There was also a missing `var`.